packages/connect-conformance/src/conformance.ts

```diff
@@ -58,31 +61,28 @@ async function download(url: string, path: string) {
   writeFileSync(path, new Uint8Array(await res.arrayBuffer()));
 }
 
-async function extractBin(path: string) {
-  if (path.endsWith(".zip")) {
-    const unzipped = unzipSync(readFileSync(path), {
+async function extractBin(archivePath: string, binPath: string) {
```

Comment:
Running the `connectconformance` command in parallel used to fail without an error message, because this script tried to write to the file that was already running.